src/Object/Api/Mastodon/Status/FriendicaExtension.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2023, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Object\Api\Mastodon\Status;
+
+use Friendica\BaseDataTransferObject;
+
+/**
+ * Class FriendicaExtension
+ *
+ * Additional fields on Mastodon Statuses for storing Friendica specific data
+ *
+ * @see https://docs.joinmastodon.org/entities/status
+ */
+class FriendicaExtension extends BaseDataTransferObject
+{
+	/** @var string */
+	protected $title;
+
+	/**
+	 * Creates a status count object
+	 *
+	 * @param string $title
+	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
+	 */
+	public function __construct(string $title)
+	{
+		$this->title = $title;
+	}
+
+	public function __get($name)
```

Comment:
This shouldn’t be needed as this object already inherits from `BaseDataTransferObject` which has a `toArray` method. It currently may not support nested objects, but I believe it could be made to do so.

Reply:
I only put that in there because I was copying another class, Counts I think, but I don't think it inherited from base. I'll remove this since it sounds redundant.

Reply:
Thanks, does it still work?